<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose_ui</class>
 <widget class="QWidget" name="Choose_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_select_red">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Agency FB&quot;;
selection-background-color: rgb(255, 0, 0);
background-color: rgb(255, 0, 0);
border-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 12pt &quot;楷体&quot;;
</string>
   </property>
   <item>
    <property name="text">
     <string>一营长的意大利炮</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北邮第三区交通委</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Void</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北邮第一交通队</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_select_blue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);
alternate-background-color: rgb(0, 255, 255);
font: 12pt &quot;楷体&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>一营长的意大利炮</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北邮第三区交通委</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Void</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北邮第一交通队</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_select_sure">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysource/picture/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_select_return">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/mysource/picture/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
